data/config-schema/src/main/java/net/corda/schema/configuration/MessagingConfig.java

```diff
@@ -78,31 +78,4 @@ private Publisher() {
      * producers to stay under this limit when publishing messages.
      */
     public static final String MAX_ALLOWED_MSG_SIZE = "maxAllowedMessageSize";
-
-    /**
-     * State Manager Configuration for connecting to the underlying persistent storage.
-     */
-    public static final class StateManager {
-        private StateManager() {
-        }
-
-        public static final String STATE_MANAGER = "stateManager";
-        public static final String TYPE = STATE_MANAGER + ".type";
-
-        // Database Values
-        public static final String DB_PROPERTIES = STATE_MANAGER + ".database";
-        public static final String JDBC_USER = DB_PROPERTIES + ".user";
-        public static final String JDBC_PASS = DB_PROPERTIES + ".pass";
-
-        public static final String JDBC_URL = DB_PROPERTIES + ".jdbc.url";
-        public static final String JDBC_DRIVER = DB_PROPERTIES + ".jdbc.driver";
-        public static final String JDBC_DRIVER_DIRECTORY = DB_PROPERTIES + ".jdbc.directory";
-        public static final String JDBC_PERSISTENCE_UNIT_NAME = DB_PROPERTIES + ".jdbc.persistenceUnitName";
```

Comment:
This could be good opportunity to get rid of the persistence unit name, we'll be removing this configurable parameter when remove hibernate use

Reply:
Removed 👍